src/rdb/rdb_raft.c

```diff
@@ -704,8 +704,21 @@ rdb_raft_exec_unpack_io(struct dss_enum_unpack_io *io, void *arg)
 		D_ASSERT(io->ui_sgls[i].sg_iovs[0].iov_len > 0);
 	}
 #endif
+	daos_epoch_t epoch = 0;
+	int i;
+
+	for (i = 0; i <= io->ui_iods_top; i++) {
```

Comment:
In lieu of the discussion on this, is there a better way to do this?

Reply:
ok, then I will use input snapshot idx as the epoch.